### PR TITLE
Translated to Hungarian!

### PR DESCRIPTION
**Pull request by ahmetlii**
Finishes #61

### CODE REVIEW

File: i18n/hu.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		"authors": [
+			"Hungarian Scratch translators", "ahmetlii"
+		]
+	},
+	"skinname-scratchwikiskin2": "ScratchWikiHéj",
+	"scratchwikiskin-desc": "Egy héj a Scratch Wiki számára.",
+	"scratchwikiskin-create": "Alkoss",
+	"scratchwikiskin-create-project": "Készítsen egy projektet",
+	"scratchwikiskin-create-page": "Hozzon létre egy projektet",
+	"scratchwikiskin-create-page-url": "Projekt:Hozzon létre egy projektet",
```
Comment (Kenny2github) [inline i18n/hu.json:12]: "Project" is a namespace name and should not be translated.
```suggestion
	"scratchwikiskin-create-page-url": "Project:Hozzon létre egy projektet",
```

File: i18n/hu.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	"scratchwikiskin-discuss-wiki-text": "Wiki Forum Téma",
+	"scratchwikiskin-ideas": "Ötletek",
+	"scratchwikiskin-ideas-text": "Scratch Ötletek",
+	"scratchwikiskin-faq-page-url": "Projekt:GYIK",
+	"scratchwikiskin-faq-page": "Wiki GYIK",
+	"scratchwikiskin-about": "Körül",
+	"scratchwikiskin-about-text": "A Scratch-ről",
+	"scratchwikiskin-helpthewiki": "Segítsen a wiki!",
+	"scratchwikiskin-madeforscratchers": "A Scratch Wikit a Scratchers készíti a Scratchers számára. Szeretne közreműködni?",
+	"scratchwikiskin-becomeacontributor-page": "Projekt:Váljon közreműködővé",
```
Comment (Kenny2github) [inline i18n/hu.json:27]:
```suggestion
	"scratchwikiskin-becomeacontributor-page": "Project:Váljon közreműködővé",
```

File: i18n/hu.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+	"scratchwikiskin-create-page-url": "Projekt:Hozzon létre egy projektet",
+	"scratchwikiskin-explore": "Böngéssz",
+	"scratchwikiskin-explore-projects": "Böngéssz Projekteket",
+	"scratchwikiskin-discuss": "Megvitat",
+	"scratchwikiskin-discuss-text": "Megvitat Scratch",
+	"scratchwikiskin-discuss-wiki": "https://scratch.mit.edu/discuss/topic/294197/",
+	"scratchwikiskin-discuss-wiki-text": "Wiki Forum Téma",
+	"scratchwikiskin-ideas": "Ötletek",
+	"scratchwikiskin-ideas-text": "Scratch Ötletek",
+	"scratchwikiskin-faq-page-url": "Projekt:GYIK",
```
Comment (Kenny2github) [inline i18n/hu.json:21]:
```suggestion
	"scratchwikiskin-faq-page-url": "Project:GYIK",
```

**Review by Kenny2github — CHANGES_REQUESTED** [review]

**Comment by ahmetlii** [comment]
@Kenny2github Done!

**Review by Kenny2github — APPROVED** [review]

**Pull request merged**